build/import/Versions.props

```diff
@@ -1,7 +1,8 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <ProjectSystemVersion>16.8.1</ProjectSystemVersion>
+    <MinimumRequiredVSVersion>16.8.1</MinimumRequiredVSVersion> <!-- We should try to keep this up to date with the highest version available on build machines -->
+    <ProjectSystemVersion>16.10.0</ProjectSystemVersion> <!-- This should always match with the VS release we're targeting -->
```

Comment:
Can you add to the comment saying it required 3 version digits? You know, just to help others in the future. :-)

Reply:
Done - definitely worth adding that note. thanks